client/src/pages/Settings/ProfilePhoto/useStyles.ts

```diff
@@ -0,0 +1,44 @@
+import { makeStyles } from '@mui/styles';
+import { Theme } from '@mui/material/styles';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  flexContainer: {
+    display: 'flex',
```

Comment:
Most of these styles can be applied directly to the `Box` component

Eg: `<Box display="flex" alignItems="center"...>`

Ditto where applicable